fastify.js

```diff
@@ -128,11 +128,11 @@ function fastify (options) {
   options.disableRequestLogging = disableRequestLogging
   options.ajv = ajvOptions
   options.clientErrorHandler = options.clientErrorHandler || defaultClientErrorHandler
-
   const initialConfig = getSecuredInitialConfig(options)
 
-  // exposeHeadRoutes have its defult set from the validatorfrom the validatorfrom the validatorfrom the validator
+  // exposeHeadRoutes and jsonShorthand have their defaults set from the validator
   options.exposeHeadRoutes = initialConfig.exposeHeadRoutes
+  options.jsonShorthand = initialConfig.jsonShorthand
```

Comment:
This is not needed thanks to the `this.initialConfig` used later on in the `route.js` file 👍🏽

Reply:
Yep, I removed it in my working version, I just haven't pushed in a while. I'll push soon.